electron-app/src/components/dashboard/ExchangeBoxNew.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <v-list-item
+    :id="`${name}_box`"
+    :to="`/exchange-balances/${name}`"
+    :ripple="false"
+    class="exchange-box__item"
+    @click="doNothing"
+  >
+    <v-list-item-avatar
+      tile
+      class="exchange-box__icon__exchange exchange-box__icon"
+    >
+      <v-img
+        contain
+        height="24"
+        :title="name"
+        :src="require(`../../assets/images/${name}.png`)"
```

Comment:
After working with vuetify/vue for more than a year I recently realized that you can use `@` here too. `@/assets` instead of hardcoding the directory structure.

Reply:
How do I rewrite this to use `@/assets` but also use a variable in the property?

Reply:
you just replace `../../` with `@` the rest remains the same :)

Reply:
Doesn't work